### ISSUE TITLE
CuDNN error: CUDNN_STATUS_SUCCESS

### ISSUE THREAD

**Issue by hzzzzjzyq** [chaitjo/graph-convnet-tsp#9]
ubuntu16.04 gpu2070s， cudnn version cant match

**Comment by chaitjo** [comment]
Hello! Thanks for your interest, this code was tested more than 1 year ago with PyTorch 0.4 and CUDA 9.0. If you are running it today, you may have to match the CUDA version or make edits to the code.

Alternatively, you may use my new codebase to train similar Non-autoregressive TSP models + you can now train on variable graph sizes instead of fixed: https://github.com/chaitjo/learning-tsp

**Comment by yzhang-gh** [comment]
I have seen the same error with Ubuntu 16.04, RTX 2080 Ti.

Seems that it can be solved by changing to CUDA 9.2.

```bash
conda remove cudnn cudatoolkit pytorch
conda install pytorch=0.4.1 cuda92 -c pytorch
```

See https://discuss.pytorch.org/t/runtimeerror-cudnn-error-cudnn-status-success/28045/5
